src/pages/get-involved/join-our-team.tsx

```diff
@@ -25,16 +48,124 @@ export default function JoinOurTeamPage({
+            </SanityImage>
+          </Grid>
+        </Grid>
+      </Section>
+      {/* Why we volunteer section */}
+      <Section backgroundColor="secondary.light">
+        <Grid container alignItems="stretch" spacing={4}>
+          <Grid xs={12}>
+            <Stack spacing={6} alignItems="center">
+              <Typography variant="h3" textAlign="center">
+                Why do we volunteer?
+              </Typography>
+              <Typography variant="h6" textAlign="center">
+                TODO: get this content from graphql query
+              </Typography>
+            </Stack>
+          </Grid>
+          {sanityJoinOurTeamPage?.volunteerImpact?.map((card) =>
+            <Grid key={card?._key} xs={12} md={4}>
+            <CardWithMedia
+              header={card?.title}
+              image={card?.image}
+              content={card?._rawDescription}
+              button={card?.button}
+              shadowColor="secondary"
+            />
+            </Grid>
+          )}
+        </Grid>
+      </Section>
+      {/* How you can get involved section */}
+      <Section backgroundColor="primary.light">
+        <Grid container alignItems="stretch" spacing={4}>
+          <Grid xs={12}>
+            <Stack spacing={6} alignItems="center">
+              <Typography variant="h3" textAlign="center">
+                How can you get involved?
+              </Typography>
+              <Typography variant="h6" textAlign="center">
+                TODO: get this content from graphql query
+              </Typography>
+            </Stack>
+          </Grid>
+          {sanityJoinOurTeamPage?.getInvolved?.map((card) =>
+            <Grid key={card?._key} xs={12} md={6}>
+            <CardWithMedia
+              header={card?.title}
+              image={card?.image}
+              content={card?._rawDescription}
+              button={card?.button}
+              shadowColor="primary"
+            />
+            </Grid>
+          )}
+        </Grid>
+      </Section>
+      {/* Volunteer interest button section */}
       <Section>
-        <Stack>
-          <Typography variant="h3">
-            {sanityJoinOurTeamPage?.mainHeader}
-          </Typography>
-          <Typography variant="h6">
-            {sanityJoinOurTeamPage?.subHeader}
+        <Stack
+          alignItems="center"
+          spacing={4}
+          justifyContent="center"
+          sx={{
+            height: 1,
+          }}>
+          <Typography variant="h4" textAlign="center">
+            {sanityJoinOurTeamPage?.interestForm}
           </Typography>
+          <SanityButton
+          isAnimated
+          boopProps={{ scale: 1.1 }}
+          content={sanityJoinOurTeamPage.interestFormButton}>
+          </SanityButton>
         </Stack>
       </Section>
+      {/* Up next section */}
+      {/* TODO: I wasn't able to find off white in the theme, we should add it? */}
```

Comment:
Yes good callout! Added a [new issue](https://github.com/TACL-LYF/lyf-website/issues/31) for that. Do you want to try and tackle that?

Reply:
You can remove this TODO and replace the background color with "offwhite". Section's [backgroundColor prop passes down to the Box component](https://github.com/TACL-LYF/lyf-website/blob/main/src/components/Layout/Section.tsx#L20), and I'm fairly confident that this will transform into "theme.palette.offwhite": https://mui.com/material-ui/react-box/#the-sx-prop